ament_mypy/ament_mypy/configuration/ament_mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
```

Comment:
I am not sure I understand the `--follow-imports` flag. Commonly when `ament_mypy` is invoked for the code of a specific package it would be good if not only the types within its code are checked but also the type information of external code if available. Though the user probably doesn't want to see error messages for code which is outside of the package (at least normally since they can't be addressed in the scope of the tested package).

Based on that I was wondering if we should add `follow_imports=silent` here?

Reply:
It probably only makes sense to do this if we start using `MYPYPATH` again. Ideally, mypy would check internal code as well as how the internal code touches the external code (and not fully check external code) but my impression right now is that it's a bit all-or-nothing: if mypy is following imports, it's type-checking it. If the type check fails, it's an error unless you use `follow_imports = silent`. The question is: Will that also silence errors regarding how internal code is using the external code? I think it does, but I need to explicitly test that case.